src/index/server.ts

```diff
@@ -208,10 +209,10 @@ export class Server extends EventEmitter {
 	 *
 	 * @example
 	 * // returns the server implementation
-	 * server.getServer().listen('8080');
+	 * server.serverImpl.listen('8080');
 	 * @returns The server implementation.
 	 */
-	public getServer() {
+	public get serverImpl() {
```

Comment:
Do we actually need to expose this anymore? It seems that we implement set,use and listen on this type. Also the name is an abbreviation.

Reply:
I believe it's a get out of jail card, in case the consumer needs to use some other express features that aren't wrapped by this library.

Reply:
For our implementations of Orizuru, exposing this is unnecessary as I've added all the functions that we've used. The `addRoute` method  should handle all the other functions such as `get`, `post` etcs. 

At this point, I don't want to remove it completely in case there are example use cases that I haven't though about.